product-lucra-gyp/web-app/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "next-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "cross-env NODE_ENV=development next dev -H 0.0.0.0 -p 3000",
+    "remote": "cross-env NODE_ENV=development NEXT_PUBLIC_API_URL=http://playrng.us-east-1.elasticbeanstalk.com next dev -H 0.0.0.0 -p 3000",
```

Comment:
This actually needs to be updated to the instance hosting our server http://playrng-lucra-gyp.us-east-1.elasticbeanstalk.com/

Reply:
https://github.com/Lucra-Sports/lucra-demo-product/pull/29/commits/1e91e1fdd84860ef930f0e73df777758b5387ee2 👍

Reply:
Disregard that commit 🙄 The two directories thing keeps throwing me off